core/shared/src/main/scala/scalaz/zio/IO.scala

```diff
@@ -830,15 +827,13 @@ object IO {
   final def require[E, A](error: E): IO[E, Option[A]] => IO[E, A] =
     (io: IO[E, Option[A]]) => io.flatMap(_.fold[IO[E, A]](IO.fail[E](error))(IO.now[A]))
 
-  final def forkAll[E, A, M[X] <: TraversableOnce[X]](
-    as: M[IO[E, A]]
-  )(implicit cbf: CanBuildFrom[M[IO[E, A]], A, M[A]]): IO[Nothing, Fiber[E, M[A]]] =
-    as.foldLeft(IO.sync[Fiber[E, mutable.Builder[A, M[A]]]](Fiber.point(cbf(as)))) { (asFiberIO, aIO) =>
-        asFiberIO.par(aIO.fork).map {
-          case (asFiber, aFiber) => asFiber.zipWith(aFiber)(_ += _)
-        }
+  final def forkAll[E, A](as: List[IO[E, A]]): IO[Nothing, Fiber[E, List[A]]] =
```

Comment:
Returning a `List` seems sensible, but there is no cost to accept a `Iterable`, which exists < 2.12 and >= 2.13.